.github/workflows/release.yaml

```diff
@@ -5,11 +5,15 @@ env:
   JAVA_VERSION: 11
   NODEJS_VERSION: 14
   PROTOBUF_VERSION: 3.10.1
-  BINTRAY_API_USER: ${{ secrets.BINTRAY_API_USER }}
-  BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+  SIGNING_KEY_ASCII_ARMORED: ${{ secrets.SIGNING_KEY_ASCII_ARMORED }}
+  SONATYPE_API_USER: ${{ secrets.SONATYPE_API_USER }}
+  SONATYPE_API_KEY: ${{ secrets.SONATYPE_API_KEY }}
 
 on:
   push:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+.x
```

Comment:
I realized that we can get snapshot builds more or less for free by just running the "release" workflow for all pushes to the `master` branch. The nexus-publish gradle plugin automatically chooses whether to publish to the OSSRH snapshot or release repository based on whether the version name includes "-SNAPSHOT".